### DISCUSSION TITLE
ComboBox executando Navigator.pop

### DISCUSSION THREAD

**Discussion by LucasWagner040** [bdlukaa/fluent_ui#586] [Q&A]
Ao clicar bem no canto de um ComboBox, o mesmo da Navigator.pop, saindo da tela, teria alguma forma de contornar este erro?

**Comment by bdlukaa** [comment]
Não consigo reproduzir

**Comment by LucasWagner040** [comment]
Por exemplo:
![Group 36](https://user-images.githubusercontent.com/74301870/197597059-81528c3e-7dee-4671-87cf-b0293b66adc0.png)

Ao clicar fora do "Item" porem não fora do ComboBox.

Ao clicar neste local, é selecionado um o item, e cai na função de "sair" do ComboBox

**Comment by LucasWagner040** [comment]
Consegui contornar este erro. Coloquei um Navigator.pop apenas quando seleciona o item. e não quando clica na ComboBox.

Obrigado pela ajuda.
